### DISCUSSION TITLE
Still no age-restricted videos fix?

### DISCUSSION THREAD

**Discussion by pozhiloy-enotik** [eritislami/evobot#1053] [Q&A]
410

**Comment by Takiyo0** [comment]
You may track for the issue in the ytdl-core's repository directly https://github.com/fent/node-ytdl-core. Here's some link that may can help you 
pull request about 410: https://github.com/fent/node-ytdl-core/pull/1010
issue about 410/403: https://github.com/fent/node-ytdl-core/issues/1021
